nucliadb_protos/nodewriter.proto

```diff
@@ -54,9 +55,19 @@ message AcceptShardRequest {
     bool override_shard = 3;
 }
 
+message NewShardRequest {
+    utils.VectorSimilarity similarity = 1;
+    string kbid = 2;
```

Comment:
NIT: why not `kbid = 1`?

Reply:
I dont know, just the order I used. Is that relevant?

Reply:
No, it's just ordering